src/relax/transform/alter_op_impl.cc

```diff
@@ -314,6 +377,8 @@ class AlterOpImplMutator : public ExprMutator {
   Map<PrimFunc, GlobalVar> cache_;
   /*! \brief Input IRModule */
   const IRModule& mod_;
+  /*! \brief Map from shape_dim attribute to the remove_pad GlobalVar */
+  Map<String, GlobalVar> remove_pad_map_;
```

Comment:
Why not `std::unordered_map<int, GlobalVar>'?

Reply:
Using `std::unordered_map<int, GlobalVar>` now.